packages/terra-tabs/src/common-tabs/_HiddenTab.jsx

```diff
@@ -84,6 +94,7 @@ const propTypes = {
 const defaultProps = {
   isSelected: false,
   showIcon: false,
+  showAddButton: false,
```

Comment:
Set default value of isClosable as false here

Reply:
Thanks ,Done https://github.com/cerner/terra-framework/commit/f4bd3b60b2862061fc633330bda690d094cd0ea6